### PR TITLE
faToEnglishNumb function + update

### PR DESCRIPTION
**Pull request by MHM5000**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
-    keywords = "example documentation tutorial",
+    license = "MIT",
+    keywords = "persian numbers charecters char keyboard-layouts ",
     url = "http://packages.python.org/persian",
     packages=['persian'],
     classifiers=[
         "Development Status :: 4 - Beta",
         "Topic :: Utilities",
-        "License :: OSI Approved :: BSD License",
+        "License :: OSI Approved :: MIT License",
```
[the remark this answers is not in the document]
Comment (itmard) [reply setup.py]: @MHM5000 Sorry my bad

[the remark this answers is not in the document]
Comment (MHM5000) [reply setup.py]: 3 yeaaaars!

I finished my military service, got married and a lot more in that time...

I believe I know you form PC7. Am I correct?

[the remark this answers is not in the document]
Comment (itmard) [reply setup.py]: @MHM5000 I am really sorry for being too late.

**Comment by itmard** [comment]
@MHM5000 can you check conflicts here ? also please update tests with your changes.
thanks!

**Pull request closed without merging**